scripts/lib/mkcloud-common.sh

```diff
@@ -117,7 +117,7 @@ function safely
 
 function rubyjsonparse
 {
-    $ruby -e "
+    ruby -e "
```

Comment:
@aplanas see commit log, I don't see a reason for hardcoding an ancient ruby version when any distro provided ruby version works just fine. this is run on the host, and newer distributions provide ruby 2.4 by default and there is just no point in requiring 2.1. the reason why that code was there originally was because there was some crappy shit version in SLE11, but we don't support SLE11 as host anymore for a long time.